docs/JavaScriptEnvironment.md

```diff
@@ -17,9 +17,13 @@ When using React Native, you're going to be running your JavaScript code in two
 While both environments are very similar, you may end up hitting some inconsistencies. We're likely going to experiment with other JS engines in the future, so it's best to avoid relying on specifics of any runtime.
 
 
-## JavaScript Transforms
+## JavaScript Syntax Transformers
 
-React Native ships with many JavaScript transforms to make writing code more enjoyable. If you are curious, you can see the [implementation of all those transformations](https://github.com/facebook/jstransform/tree/master/visitors). Here's the full list:
+Syntax transformers make writing code more enjoyable by allowing you to use new JavaScript syntax without having to wait for support on all interpreters.
```

Comment:
We don't *really* have to wait on all interpreters. We can just end the sentence at `by allowing you to use new JavaScript language features`.